test/integration/kudo-upgrade-command/01-assert.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: kudo.dev/v1alpha1
+kind: Instance
+metadata:
+  name: upgrade-instance
+spec:
+  operatorVersion:
+    name: upgrade-operator-0.2.0
+  parameters:
+    replicas: 4
```

```suggestion
    replicas: "4"
```

This is needed to fix the test. But is it desirable to set the replica as a string?